google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/OptionTest.java

```diff
@@ -57,7 +54,11 @@ public void testConstructor() {
     Option option = new Option(RPC_OPTION, null) {};
     assertEquals(RPC_OPTION, option.getRpcOption());
     assertNull(option.getValue());
-    thrown.expect(NullPointerException.class);
-    new Option(null, VALUE) {};
+    try {
+      new Option(null, VALUE) {};
+      Assert.fail();
+    } catch (NullPointerException ex) {
+      assertNull(ex.getMessage());
```

Comment:
I suppose this is current behavior, but there should be a message here and I'd prefer not to lock in its nonexistence with this assert.